os/pm/pm_metrics.c

```diff
@@ -80,6 +91,17 @@ static void pm_print_metrics(double total_time, int n_domains)
 	for (index = 0; index < PM_WAKEUP_SRC_COUNT; index++) {
 		pmdbg(" %14s | %6d \n", wakeup_src_name[index], g_pm_metrics->wakeup_src_counts[index]);
 	}
+	if (g_pm_metrics->wakeup_src_counts[PM_WAKEUP_HW_TIMER]) {
+		pmdbg("\n");
```

Comment:
How about adding new definition for pmdbg_noarg (based on dbg_noarg) and use that to print these mertrics? When we are printing a table, then we dont need the api name at the beginning of each line of the table.

Reply:
Done